Readme_Direct_Ethernet.txt

```diff
@@ -0,0 +1,11 @@
+The direct ethernet is no longer being actively developed here. To
+avoid confusion it has been removed from master.
+
+The latest binary release (1.2.3) is available via bintray:
+
+https://bintray.com/labrad/generic/direct-ethernet
+
+The source code is available in a release tag of the git repository:
```

Comment:
Here, can just say that the latest release is at tag `direct-ethernet-v1.2.3`. Can give the github link as well, but people using git command line tools just need to know the tag name.